### PR TITLE
:book: Add blog release announcement for v1.5

### PR DESCRIPTION
**Pull request by mudler**
Signed-off-by: mudler <mudler@c3os.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
----
-name: 'Question/Support'
-about: 'Ask a question about kairos or request support.'
```
Comment (mudler) [inline .github/ISSUE_TEMPLATE/question.md:3]: we have a link to discussions already, no need a template for it

File: docs/content/en/blog/release-v1.5.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## OS updates
+
+We've added RockyLinux and Debian to our list of supported releases. This means you can now run Kairos on both and take advantage of their features and stability.
+
+We've also updated our Alpine support, so you can now run Kairos on the latest version of Alpine Linux.
+
+## Extend Kairos
+
+Extend the configuration of your node with custom, container-based deployment models (see [our documentation here](/docs/advanced/bundles) and a [full example showing how to deploy MetaLB](/docs/examples/bundles)). There are also available `Kubevirt` and `MetalLB` bundles from the [community-bundles](https://github.com/kairos-io/community-bundles) repository.
+
```
Comment (oz123) [inline docs/content/en/blog/release-v1.5.md:46]: Suggestion:

```
`Kubevirt` and `MetalLB` bundles are availble in the [community-bundles](https://github.com/kairos-io/community-bundles) repository.
```

Comment (mudler) [reply docs/content/en/blog/release-v1.5.md:46]: :+1: fixed in https://github.com/kairos-io/kairos/pull/750/commits/a98b7307eae0767b4f621649af4ffd6fbb52ac94

**Review by Itxaka — APPROVED** [review]

**Review by oz123 — DISMISSED** [review]

File: docs/content/en/blog/release-v1.5.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## Protect Your Data with User Data Encryption at the Edge
+
+Kairos 1.5 now allows you to encrypt your user data with ease, keeping it secure from prying eyes. Encryption is done via TPM and optionally with the Kairos KMS (Key Management Server) for external authentication and management of encrypted secrets. Check out our [documentation](/docs/advanced/partition_encryption) for more information on partition encryption.
+
+## OS updates
+
+We've added RockyLinux and Debian to our list of supported releases, giving you more options to run Kairos on both stable and feature-rich operating systems. We've also updated our Alpine support, so you can now run Kairos on the latest version of Alpine Linux.
+
+## Extend Kairos with Custom Deployment Models
```
Comment (Itxaka) [inline docs/content/en/blog/release-v1.5.md:35]: Bundles? or do we like the new name that match :P

Comment (mudler) [reply docs/content/en/blog/release-v1.5.md:35]: :dagger:

**Review by Itxaka — APPROVED** [review]

**Pull request merged**